code/src/Pages/Home.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { tasks } from '../Reducers/tasks'
+
+import { List } from '../Components/List'
+import { Header } from '../Components/Header'
+import { UserInput } from '../Components/UserInput'
+import { Button } from '../library/Button'
+import { Main, Wrapper, OuterWrapper } from '../library/Containers'
+
+export const Home = () => {
+  const clearBtn = true
```

Comment:
I don't see a reason for this variable as well, you could just pass boolean value to component itself, no need to wrap it in variable if we don't overwrite it anywhere